requirements.txt

```diff
@@ -1,5 +1,5 @@
-pandas==1.3.1
-snakemake==6.8.0
-singularity==3.0.0
+hydra-genetics==0.9.0
```

Comment:
0.9.2?

Reply:
:man_facepalming: I will make a pull-request for that, before creating the new release